### ISSUE TITLE
Double Placement to enable Pstone

### ISSUE THREAD

**Issue by EvilYodeler** [marcelo-mason/PreciousStones#968]
This may be the issue that Frizzbee is talking about, but I wanted to include a video to show the specifics.
https://youtu.be/wZ1XH0xjhoc

Thank you for your time and an amazing plug-in!

**Comment by marcelo-mason** [comment]
Great vid, helped A LOT.

**Comment by frizzbee30** [comment]
yep, that's the issue I was having, looking forward to trying the new build. Thanks

**Comment by EvilYodeler** [comment]
Thank you for the quick response much appreciated. How do I donate to your development? I got an error when trying to donate through the Bukkit donate button on your page there.

**Comment by krustymk** [comment]
Sorry for the off topic but i noticed you have tool tips in the video.. I dont get that, how do i enable it?

Found the answer when i generated a new config. 

Meta tags not added to old configs

for example

  meta-name: '&bCity Protection'
  meta-autoset: true
  meta-lore:
- '&eThis will protect a city'
- '&eIt covers an area of 20 radius'
- '&eWith a max height of 15'

would be missing

Cant find this documented
